src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html

```diff
@@ -15,10 +15,12 @@
 -->
 
 <md-content layout="row" layout-fill flex>
-  <div class="kd-replicationcontrollerdetail-sidebar" layout="column" flex="noshrink">
+  <div class="kd-replicationcontrollerdetail-sidebar" layout="column" flex="noshrink"
+       ng-show="ctrl.mdMedia('gt-sm')">
```

Comment:
Btw, move all `ctrl.mdMedia('gt-sm')` to the ctrl. Otherwise this will most likely be broken when somebody changes the line here, but not 37.